content/information-distribution/node.md

```diff
@@ -44,81 +44,129 @@ This ping **must** take the form of a Matrix timeline event of the
 
 An entry node is a node that allows client implementations to connect to the
 federations this node is in, and retrieve information from them. An entry node
-**must** allow guest access, and **must** define at least one
-[alias](https://matrix.org/docs/spec/client_server/r0.4.0.html#room-aliases) to
-the Matrix room.
-
-Client implementations **must** provide users with an updated list of entry
-nodes with their addresses and the aliases to use to join the federation the
-node is in. Clients implementations **must** also provide users with a way to
-input custom values for the address of the entry node to use and the alias of
-the Matrix room to join.
-
-Once a Matrix room is joined, client implementations **must** retrieve and save
-the [list of
-aliases](https://matrix.org/docs/spec/client_server/r0.4.0.html#m-room-aliases)
-for this Matrix room. This can be done by processing all of the `m.room.aliases`
-events in the state of the Matrix room, each of them containing the list of
-aliases defined for a given node. The content for such an event looks like this:
+**must** allow guest access.
+
+On top of that, an entry node **must** expose a file at the address
+`https://SERVER_NAME/.well-known/informo/info` which content is a map in the
+JSON format, which associates Matrix room IDs with a map implementing the
+following structure:
```

Comment:
Setting `entry_point` to `false` for a federation already works as a way to blacklist federations.

Reply:
Conclusions from private chat: we'll need to couple that with a `*` wildcard value for the federation which would allow one to define either a blacklist of a whitelist of entry nodes with the same content format.